.github/workflows/deploy-review.yml

```diff
@@ -0,0 +1,143 @@
+name: Deploy Review App on PR
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+permissions:
+  id-token: write
+  contents: read
+  packages: write
+
+concurrency:
+  group: ${{ github.event.pull_request.number }}
+
+jobs:
+  check-secret-access:
+    runs-on: ubuntu-latest
+    outputs:
+      allowed: ${{ steps.check-secret.outputs.allowed }}
+    steps:
+      - name: Test for secrets access
+        id: check-secret
+        shell: bash
+        run: |
+          unset allowed
+          if [ "${{ secrets.TEST_SECRETS_ACCESS }}" != '' ]; then
+            echo "allowed=true" >> $GITHUB_OUTPUT;
+          else
+            echo "allowed=false" >> $GITHUB_OUTPUT;
+          fi
+
+  build:
+    runs-on: ubuntu-latest
+    container: node:18-alpine
+    steps:
+      - name: 'Install deps'
+        run: |
+          apk add python3 make g++ git
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: 'npm install and build'
+        run: |
+          npm install
+          npm ci
+          npm run build
+      - name: Upload build to artifacts
+        uses: actions/upload-artifact@v3
+        with:
+          name: build
+          path: build
+
+  package:
+    runs-on: ubuntu-latest
+    needs:
+      - build
+      - check-secret-access
+    if: needs.check-secret-access.outputs.allowed == 'true'
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Download build from artifacts
+        uses: actions/download-artifact@v3
+        with:
+          name: build
+          path: ./build
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to GitHub Container Registry
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ github.token }}
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ghcr.io/${{ github.repository }}
+          tags: |
+            type=sha,format=long
+      - name: Build Docker Container
+        uses: docker/build-push-action@v3
+        with:
+          context: .
```

Comment:
Need a Dockerfile at the root of this repo

Reply:
The one from chia-docs should just work since we build the app outside of the Dockerfile first:
```
FROM nginx:stable-alpine
COPY build /usr/share/nginx/html
EXPOSE 80
CMD nginx -g "daemon off;"
```